Core/clim/input-editing-drei.lisp

```diff
@@ -34,6 +34,12 @@ Members of this class are mutable."))
 (defmethod interactive-stream-p ((stream standard-input-editing-stream))
   t)
 
+(defmethod port ((stream standard-input-editing-stream))
+  (port (encapsulating-stream-stream stream)))
+
+(defmethod graft ((stream standard-input-editing-stream))
+  (graft (encapsulating-stream-stream stream)))
+
```

Comment:
perhaps these should be

(def-stream-method …) in `encapsulate.lisp` for the standard-encapsulating-stream?

Reply:
done.